.gitignore

```diff
@@ -57,6 +57,7 @@
 
 # ddev
 .ddev/config.yaml
+.ddev/.sampleData
```

Comment:
wouldn't it be possible to just have .ddev, removing the whole folder from git, is it needed for something else?

Reply:
I would not exclude this directory from git because it is a test environment, changes can be made in it and would be good to know about. Thanks to your comment I noticed that the DDEV configuration file is excluded and I don't think it's a good idea. I would ignore only the directory where the archive with the Magento Sample package was downloaded.